src/appmixer/line/core/LeaveGrouporRoom/component.json

```diff
@@ -0,0 +1,66 @@
+{
+    "name": "appmixer.line.core.LeaveGrouporRoom",
```

Comment:
please rename, what is "Groupor"  ? :) => LeaveGroupOrRoom
![image](https://github.com/user-attachments/assets/7bdc12e0-2a80-4c5b-a88d-cc0f2b09a4e5)